src/libraries/System.IO.FileSystem/tests/RandomAccess/NoBuffering.Windows.cs

```diff
@@ -242,6 +242,80 @@ public async Task ReadShouldReturnZeroForEndOfFile(bool asyncOperation, bool asy
             Assert.Equal(fileSize, total);
         }
 
+        [Fact]
+        public void ReadFromDiskAfterFlushToDisk()
+        {
+            // Save test file path so we can refer to the same file throughout the test.
```

Comment:
```suggestion
```
keep comments to non obvious :)

Reply:
I hear you :)

I once worked on a project where every line of code was commented, even the tests! Coming from the "zero comments unless absolutely necessary" camp, I found this practice really perplexing so I asked the team leader why everyone on the team did this. **I remember his response word for word because it profoundly changed the way I write code**. He said:

> Document everything. If it is not already obvious to the reader, then the reader will gain understanding. If it is already obvious to the reader, then the reader will be reassured that their understanding is correct.

So to this day I generally follow the "document everything" approach to commenting and it has been made even easier to do by GitHub CoPilot which will happily write much of the code for me when I document even the obvious stuff.

In any case, I have followed your recommendation and removed many of the non-essential comments in [this commit](https://github.com/dotnet/runtime/pull/89100/commits/a7d6289990c02f84d6d2b79fbd7959ded2ceee74). Some comments remain and I hope they add value but if not, please let me know and I will be happy to remove all of them 👍

Reply:
Yeah, it's a matter of taste to some extent. Too many comments can make code less readable just like too few comments does. Good naming, structure etc can sometimes do the job.